bitnami/redis-cluster/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 annotations:
   category: Database
 apiVersion: v2
-appVersion: 6.0.10
+appVersion: 6.0.11
```

Comment:
This is the application version (the redis one), the chart version is the `version` field at the end of this file. I am going to edit the file by myself and it should be ready to be landed

Reply:
Yep sorry, I did this too quickly
Thanks!